backend/app/entity/shortlink.go

```diff
@@ -17,3 +17,10 @@ type ShortLink struct {
 	OpenGraphTags metatag.OpenGraph
 	TwitterTags   metatag.Twitter
 }
+
+// ShortLinkInput represents possible ShortLink attributes for a new short link or an updated short link.
+type ShortLinkInput struct {
+	LongLink    string
+	CustomAlias string
```

Comment:
We need to allow those value to be nullable so that the use case and repository can interpret their meaning themselves. Without nil, there is no clear separation between values not set and set wrong value.

Reply:
When we update the short link, we will ignore the values if they are nil. If not, we override it with empty string for long link. There is no way to distinguish between those.

Reply:
What are your thoughts on this?

Reply:
As discussed on Slack, we have decided on passing nullable fields into use case, and use case decides what to do with it.